tests/test_ai_tools.py

```diff
@@ -123,12 +123,12 @@ def test_similar_value_tool_e2e(self):
         agent = spark_ai._create_sql_agent()
         similar_value_tool = agent.lookup_tool("similar_value")
 
-        table_file = "tests/data/test_similar_value_tool_e2e.tables.jsonl"
+        table_file = "tests/data/test_transform.tables.jsonl"
         source_file = "tests/data/test_similar_value_tool_e2e.jsonl"
 
         # prepare tables
         statements = create_temp_view_statements(table_file)
-        for stmt in statements:
+        for stmt in statements[:3]:
```

Comment:
This is too hacky. It's ok to have two jsonl files, one for test_ai_tools and another one for end_to_end tests.
This way, our tests are easier to understand.